vademecum_VT.md

```diff
@@ -47,13 +54,19 @@ Dalla versione Quantum, Firefox integra il nuovo sistema Style che velocizza il
 Questa tecnologia è parte del nuovo browser engine Servo ed è presente anche sulle versioni per dispositivi mobile, ovvero su Android e iOS.  
 
 #### WebVR |QRcode|
-Esplora la realtà virtuale direttamente da Firefox, senza necessità di installare software specifici. Con WebVR la realtà aumentata e virtuale è gratuita, semplice e per tutti.
+Firefox supporta lo standard _WebVR_, che permette di avere la realtà virtuale direttamente sul browser in tutte le piattaforme.
+Mozilla ha sviluppato il framework open source _A-Frame_ che, utizzando i web component e three.js, permette di scrivere le scene 3D direttamente in html. In questo modo chiunque può creare un ambiente 3D, mettere un video 3D. Inoltre A-Frame ha diversi sottoprogetti come A-Terrain, per visualizzare dati geografici, A-Blast, per creare giochi con WebVr, A-Painter, per disegnare in ambiente 3D.
```

Comment:
"In questo modo chiunque può creare un ambiente 3D, mettere un video 3D." è una frase piazzata lì senza alcun collegamento. Anzi, è collegata ma incompleta.